addons/project/models/res_config_settings.py

```diff
@@ -13,6 +13,7 @@ class ResConfigSettings(models.TransientModel):
     group_project_rating = fields.Boolean("Customer Ratings", implied_group='project.group_project_rating')
     group_project_recurring_tasks = fields.Boolean("Recurring Tasks", implied_group="project.group_project_recurring_tasks")
     group_project_task_dependencies = fields.Boolean("Task Dependencies", implied_group="project.group_project_task_dependencies")
+    group_analytic_accounting = fields.Boolean(string='Analytic Accounting', implied_group='analytic.group_analytic_accounting')
```

Comment:
Project depends on analytics, the field is already existing from https://github.com/odoo/odoo/pull/72736/files#diff-805abbae908ebd378ebfbc62487f277027148a56b5f2186896ed6fa09d8b5b56R9

This can be removed.